doc/classes/UDPServer.xml

```diff
@@ -7,7 +7,8 @@
 		A simple server that opens a UDP socket and returns connected [PacketPeerUDP] upon receiving new packets. See also [method PacketPeerUDP.connect_to_host].
 		After starting the server ([method listen]), you will need to [method poll] it at regular intervals (e.g. inside [method Node._process]) for it to process new packets, delivering them to the appropriate [PacketPeerUDP], and taking new connections.
 		Below a small example of how it can be used:
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		# server.gd
 		extends Node
```

Comment:
```suggestion
		class_name Server
		extends Node
```
We should use `class_name` instead of the comments. I have to define a class name in C# either way and it is a good practice IMO.